api/v1/campaigns.coffee

```diff
@@ -0,0 +1,29 @@
+ObjectID = require('mongodb').ObjectID
+
+module.exports = (api, db) ->
+  api = api || {}
+  api.v1 = api.v1 || {}
+
+  api.v1.getCampaign = (req, res) ->
+    db.collection("campaigns").find( {_id : req.params.id} ).toArray (err, results) ->

```

Comment:
Yeah, let's change this to use `findOne` so we just get one document. If you can't get it working let me know and I'll give it a try.